src/actions/actionGroup.tsx

```diff
@@ -232,9 +233,18 @@ export const actionUngroup = register({
     });
 
     // remove binded text elements from selection
-    boundTextElementIds.forEach(
-      (id) => (updateAppState.selectedElementIds[id] = false),
+    updateAppState.selectedElementIds = Object.fromEntries(
+      Object.entries(updateAppState.selectedElementIds).reduce(
+        (acc: [ExcalidrawElement["id"], true][], [id, selected]) => {
+          if (selected && !boundTextElementIds.includes(id)) {
+            acc.push([id, true]);
```

Comment:
Is it necessary to use  `Object.fromEntries(Object.entries` if could be already an object?

Reply:
yeah, I wasn't thinking